custom_components/waste_collection_schedule/waste_collection_schedule/source/lisburn_castlereagh_gov_uk.py

```diff
@@ -0,0 +1,100 @@
+import requests
+import bs4
+import difflib
+
+from datetime import datetime
+
+from waste_collection_schedule import Collection
+
+TITLE = "Lisburn and Castlereagh City Council" # Title will show up in README.md and info.md
+DESCRIPTION = "Source for the Lisburn and Castlereagh City Council"  # Describe your source
+URL = "https://lisburncastlereagh.gov.uk/"  # Insert url to service homepage. URL will show up in README.md and info.md
+TEST_CASES = {
+    "Test_001": {"postcode": "BT28 1AG", "house_number": "19A"},
+    "Test_002": {"postcode": "BT26 6AB", "house_number": "31"},
```

Comment:
Your Source should also work with integer `house_number` the user should not care about how to format the number. 
Please add a Test for an integer house_number.

Reply:
Good point. Have updated handling of the house number and added a test.

Reply:
New testing output:
```❯ ./test_sources.py -s lisburn_castlereagh_gov_uk -i -l                        
Testing source lisburn_castlereagh_gov_uk ...
  found 5 entries for Test_001
    2023-05-03 : Recycling [mdi:recycle]
    2023-05-03 : Garden [mdi:leaf]
    2023-05-10 : Refuse [mdi:trash-can]
    2023-05-17 : Recycling [mdi:recycle]
    2023-05-17 : Garden [mdi:leaf]
  found 4 entries for Test_002
    2023-05-01 : Refuse [mdi:trash-can]
    2023-05-08 : Recycling [mdi:recycle]
    2023-05-08 : Garden [mdi:leaf]
    2023-05-15 : Refuse [mdi:trash-can]
  found 4 entries for Test_003
    2023-05-01 : Refuse [mdi:trash-can]
    2023-05-08 : Recycling [mdi:recycle]
    2023-05-08 : Garden [mdi:leaf]
    2023-05-15 : Refuse [mdi:trash-can]
  found 4 entries for Test_004
    2023-05-01 : Refuse [mdi:trash-can]
    2023-05-08 : Recycling [mdi:recycle]
    2023-05-08 : Garden [mdi:leaf]
    2023-05-15 : Refuse [mdi:trash-can]
  found 5 entries for Test_005
    2023-05-05 : Recycling [mdi:recycle]
    2023-05-05 : Garden [mdi:leaf]
    2023-05-12 : Refuse [mdi:trash-can]
    2023-05-19 : Recycling [mdi:recycle]
    2023-05-19 : Garden [mdi:leaf]```